packages/grafana-flamegraph/src/FlameGraphContainer.tsx

```diff
@@ -234,25 +255,55 @@ function useColorScheme(dataContainer: FlameGraphDataContainer | undefined) {
   return [colorScheme, setColorScheme] as const;
 }
 
-function getStyles(theme: GrafanaTheme2, vertical?: boolean) {
+function getStyles(theme: GrafanaTheme2) {
   return {
     container: css({
       label: 'container',
+      overflow: 'auto',
       height: '100%',
-      display: vertical ? 'block' : 'flex',
+      display: 'flex',
       flex: '1 1 0',
       flexDirection: 'column',
       minHeight: 0,
       gap: theme.spacing(1),
     }),
     body: css({
       label: 'body',
-      display: 'flex',
       flexGrow: 1,
+    }),
+
+    tableContainer: css({
+      // This is not ideal for dashboard panel where it creates a double scroll. In a panel it should be 100% but then
+      // in explore we need a specific height.
+      height: 800,
```

Comment:
Yeah don't know at this moment. It's sort of similar to trace view issue we talked about and for now I would probably keep it as it is think about the overall pattern. We may end up with needing to know exactly what context visualization is running in but then it's possible for them to drift too much between context. Maybe it would be easier to make explore a bit more like dashboard in the sense that it should set a size for the visualization as a panel and that way it would converge more reasonably. Again not sure right now and the double scroll probably isn't that pressing issue right now.